doc/overview/deegree_overview.rst

```diff
@@ -122,7 +126,4 @@ Details
 
 
 @VMDK_deegree@
-@QUICKSTART_deegree@
-
-.. presentation-note
```

Comment:
Looks like the note for the [presentation](https://osgeo.github.io/OSGeoLive-doc/en/presentation.html#/7/1) was deleted: (Press "s" and the notes will pop out in a different window)

Reply:
![image](https://user-images.githubusercontent.com/5035290/178125339-3959131f-1b89-44c7-8086-27d6b20456f4.png)